pkg/webhook/webhook.go

```diff
@@ -17,9 +17,12 @@ package webhook
 import (
 	"context"
 	"crypto/x509"
+	"fmt"
+	"html/template"
```

Comment:
Did you on purpose choose `html` instead of `text`? `html/template` seems to be [mainly recommended for HTML templates](https://cs.opensource.google/go/go/+/refs/tags/go1.20.3:src/html/template/doc.go;l=7-9), so we could also use `text/template`.
```suggestion
	"text/template"
```

Reply:
thanks